<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1201</width>
     <height>721</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1201</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="2" column="1">
       <widget class="QPushButton" name="moveToGraph">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:0.523, x2:0, y2:0.494, stop:0 rgba(0, 0, 0, 255), stop:0.897727 rgba(57, 57, 57, 255))</string>
        </property>
        <property name="text">
         <string>View Map Traversals</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>300</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>200</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="moveToEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:qlineargradient(spread:pad, x1:0, y1:0.506, x2:1, y2:0.511045, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(52, 52, 52, 255))</string>
        </property>
        <property name="text">
         <string>Edit Database</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="moveToTripSelect">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:qlineargradient(spread:pad, x1:0, y1:0.506, x2:1, y2:0.511045, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(52, 52, 52, 255))</string>
        </property>
        <property name="text">
         <string>Plan a Vacation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>300</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="moveToView">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:0.523, x2:0, y2:0.494, stop:0 rgba(0, 0, 0, 255), stop:0.897727 rgba(57, 57, 57, 255))</string>
        </property>
        <property name="text">
         <string>View NBA Information</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>200</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="mainButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:0.523, x2:0, y2:0.494, stop:0 rgba(0, 0, 0, 255), stop:0.897727 rgba(57, 57, 57, 255))</string>
      </property>
      <property name="text">
       <string>Main
Menu</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="titleText">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>NBA Basketball Vacation Planner</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color:qlineargradient(spread:pad, x1:0, y1:0.506, x2:1, y2:0.511045, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(52, 52, 52, 255))</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
